.github/workflows/ci.yaml

```diff
@@ -25,11 +24,14 @@ env:
 
 jobs:
   R:
-    runs-on: windows-latest
+    runs-on: [self-hosted, 1ES.Pool=sqlmlutils_GH_RunnerPool]
     strategy:
-      fail-fast: false
       matrix:
-        r-version: ["3.5.2", "4.2"]
+        include:
+          - r-version: "3.5.2"
+            sql-platform: "box"
+          - r-version: "4.2.0"
+            sql-platform: "box"
```

Comment:
Right now it is box only, but this will include MI in the future, correct?

Reply:
Correct, that is a follow up once that infrastructure is configured for use here.